client/src/components/Collections/common/CollectionEditView.vue

```diff
@@ -0,0 +1,260 @@
+<template>
+    <div>
+        <h4>{{ l("Edit Collection Attributes") }}</h4>
+        <b-alert show variant="warning" dismissible>
+            {{
+                l(
+                    "Collections are immutable. This means there will be some things you cannot change without creating a new collection. "
+                )
+            }}
+        </b-alert>
+        <div v-if="jobError">
+            <b-alert show variant="danger" dismissible>
+                {{ errorMessage }}
+            </b-alert>
+        </div>
+        <b-tabs content-class="mt-3">
+            <b-tab>
+                <template v-slot:title> <i class="fa fa-bars"></i>{{ l(" Attributes") }}</template>
```

Comment:
Can these use the more modern fontawesome components instead of icons?  We want to get rid of class-based fontawesome icons wherever we can.